src/workers/notifications/postMention.ts

```diff
@@ -0,0 +1,39 @@
+import { messageToJson } from '../worker';
+import { PostMention, User } from '../../entity';
+import {
+  NotificationDoneByContext,
+  NotificationPostContext,
+} from '../../notifications';
+import { NotificationWorker } from './worker';
+import { ChangeObject } from '../../types';
+import { buildPostContext } from './utils';
+
+interface Data {
+  postMention: ChangeObject<PostMention>;
+}
+
+const worker: NotificationWorker = {
+  subscription: 'api.comment-mention-notification',
```

Comment:
Seems like the wrong name. Please also make sure to add this new worker to the index file and pulumi